.travis.yml

```diff
@@ -5,15 +5,33 @@ env:
 install:
   - pip install GitPython
   - export TRAVIS_COMMIT_MSG="$TRAVIS_REPO_SLUG - $(git log --format=%B --no-merges -n 1)"
-  - git clone https://github.com/xbmc-catchuptv-au/repo-devel.git .deploy
 script:
-  - cd .deploy
-  - ./manage_repo.py ../
+  - >
+    git clone https://github.com/xbmc-catchuptv-au/repo-devel.git .deploy && 
+    cd .deploy && 
+    ./manage_repo.py ../ || travis_terminate 1
+  - |
+    if [ -n "$TRAVIS_TAG" ]; then
+        cd ..
+        git clone https://github.com/xbmc-catchuptv-au/repo.git .deploy-prod && \
+        cd .deploy-prod && \
+        ./manage_repo.py ../ || travis_terminate 1
+    fi
 after_script:
   - git config --global user.email 'aussieaddons@aussieaddons.com'
   - git config --global user.name 'Aussie Add-ons Bot'
+  - cd ../.deploy
   - git config credential.helper "store --file=.git/credentials"
   - echo "https://${GH_TOKEN}:@github.com" > .git/credentials
   - git add .
   - git commit --allow-empty -m "$TRAVIS_COMMIT_MSG"
   - git push
+  - |
+    if [ -n "$TRAVIS_TAG" ]; then
+        cd ../.deploy-prod
+        git config credential.helper "store --file=.git/credentials"
+        echo "https://${GH_TOKEN}:@github.com" > .git/credentials
+        git add .
```

Comment:
Is it worth adding travis_terminate 1 to these lines as well? Any benefit?

`git add . && git commit --allow-empty -m "Update $(basename `git -C ../ rev-parse --show-toplevel`) to $TRAVIS_TAG && git push || travis_terminate 1`

Reply:
Yep for sure, I can also remove it from line 12.